image/image.go

```diff
@@ -58,44 +57,39 @@ func closeResponseBody(ctx context.Context, resp *http.Response) {
 	}
 }
 
-// NewAPIClient creates a new instance of Client with a given image api url and the relevant tokens
+// NewAPIClient creates a new instance of ImageAPI Client with a given image api url
 func NewAPIClient(imageAPIURL string) *Client {
-	hcClient := healthcheck.NewClient(service, imageAPIURL)
-
 	return &Client{
-		cli: hcClient.Client,
-		url: imageAPIURL,
+		healthcheck.NewClient(service, imageAPIURL),
```

Comment:
I think this is why I thought it odd to have the option to pass in the healthcheck. Would it be better to always initialise the healthcheck client internally to the client? Then instead of creating the healthcheck client to pass in, you would create and pass in the clienter.